fastlane/Fastfile

```diff
@@ -244,7 +247,6 @@ platform :ios do
 
     # Wrap up
     version = ios_get_app_version
-    # ios_clear_intermediate_tags(version: version)
```

Comment:
Apart from the fact this is a line of commented code, so whether it's there or deleted the result is the same, you chose to remove this because we've agreed to not remove the intermediate tags, right? So we're in line with Android etc.

Reply:
Yep exactly.

I think this line got commented a while ago when there was the bug that deleted all the tags accidentally, and never got un-commented once the fix shipped in toolkit apparently.

But since now we decided to create the GH releases for intermediate betas like for Android, we agreed to keep the tags too so that the GitHub releases don't go back from published state back to draft state if we deleted the tags. Hence no more tag deletion and removing this line once and for all